packages/synapse-react-client/src/mocks/entity/mockProject.ts

```diff
@@ -20,6 +21,19 @@ export const mockProjectIds = times(20).map(i => i + 10001)
 const MOCK_PROJECT_ID = `syn10000`
 const MOCK_PROJECT_NAME = 'A Mock Project'
 
+export const mockDoiAssociation: DoiAssociation = {
```

Comment:
Move into an exported mock, which can be used to resolve lint errors in TitleBar integration tests